Script/TypeScript/AtomicExtended.d.ts

```diff
@@ -0,0 +1,115 @@
+// Manual typescript definitions from various CPP and JS files that can't yet be autogenerated
+// no-default-lib and modified lib.d.ts used to allow redefinition of self
+/// <reference no-default-lib="true"/>
+/// <reference path="./lib.d.ts" />
```

Comment:
Sorry I just removed that, missed the ref. Will nuke it now push


Regards,
Matt Benic
mattbenic@gmail.com | +27 (72) 1826161 | za.linkedin.com/in/mattbenic

On Wed, Aug 12, 2015 at 2:43 PM, JoshEngebretson <notifications@github.com>
wrote:

> In Script/TypeScript/AtomicExtended.d.ts
> <https://github.com/AtomicGameEngine/AtomicGameEngine/pull/178#discussion_r36854639>
> :
>
> > @@ -0,0 +1,115 @@
> > +// Manual typescript definitions from various CPP and JS files that can't yet be autogenerated
> > +// no-default-lib and modified lib.d.ts used to allow redefinition of self
> > +/// <reference no-default-lib="true"/>
> > +/// <reference path="./lib.d.ts" />
>
> Does this require a custom lib.d.ts? I am not seeing it here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/AtomicGameEngine/AtomicGameEngine/pull/178/files#r36854639>
> .
>